DESCRIPTION

```diff
@@ -39,9 +40,11 @@ Imports:
     airway,
     org.Hs.eg.db,
     org.Dm.eg.db,
+    AnnotationDbi,
```

Comment:
AnnotationDbi comes automatically with org packages afaik but is is better to add it explicitly?

Reply:
It was giving me error CHECK locally, that's why I added it.

Reply:
Ok fair enough.